frameworks/PHP/php-silex/setup_raw.sh

```diff
@@ -1,6 +1,10 @@
 #!/bin/bash
+export PHP_HOME=${IROOT}/php-5.5.17
+export PHP_FPM=$PHP_HOME/sbin/php-fpm
+export COMPOSER_HOME=${IROOT}/php-composer
+export NGINX_HOME=${IROOT}/nginx
 
-sed -i 's|192.168.100.102|'"${DBHOST}"'|g' web/index_raw.php
+sed -i 's|172.16.98.120|'"${DBHOST}"'|g' web/index_raw.php
```

Comment:
This should probably be changed to a regular expression that doesn't rely on whatever IP is in the file.